src/index.ts

```diff
@@ -44,3 +44,4 @@ export { StatusEnum } from './models/statusEnum';
 export { Transcript } from './models/transcript';
 export { Transcription } from './models/transcription';
 export { TranscriptionResponse } from './models/transcriptionResponse';
+export * from './bxml'
```

Comment:
We need to figure out how to either skip this file when updating the generated code or to append this line by shell to this file.

Reply:
oof yeah good callout, because this is both going to be autogenerated and have custom code from us right? I don't want to rely on manual PR checks to make sure this line stays there

Reply:
Yea exactly, I tried to think of another way of exporting the bxml but it needs to be in this file :/

Reply:
For now we can make sure to have bxml unit tests that will catch if we accidentally remove it, but yeah long term we'll need a better solution